src/SQLite.cs

```diff
@@ -2817,23 +2817,54 @@ public IEnumerable<T> ExecuteDeferredQuery<T> (TableMapping map)
 				_conn.Tracer?.Invoke ("Executing Query: " + this);
 			}
 
+			var values = new Dictionary<int, Tuple<string, string>> ();
 			var stmt = Prepare ();
 			try {
 				var cols = new TableMapping.Column[SQLite3.ColumnCount (stmt)];
 
 				for (int i = 0; i < cols.Length; i++) {
 					var name = SQLite3.ColumnName16 (stmt, i);
-					cols[i] = map.FindColumn (name);
+					if (name.Contains (".")) {
+						var properties = name.Split ('.');
+						if (properties.Length == 2)
```

Comment:
Only supports properties that are nested 1 deep?

Reply:
Yes. There was a good reason for that but I can't remember it now.